graph/resolver/discussion.resolvers.go

```diff
@@ -34,6 +35,22 @@ func (r *discussionResolver) Posts(ctx context.Context, obj *model.Discussion) (
 	return posts, nil
 }
 
+func (r *discussionResolver) PostsConnection(ctx context.Context, obj *model.Discussion, after *string) (*model.PostsConnection, error) {
+	/* Hardcode the number of posts per page. This can be changed to be settable by the client in the query. */
+	limit := 50
+
+	/* Sanity check. If no "after" parameter is specified, we set it to a time far into the future, for which
```

Comment:
Cursor creation is explicitly handled in the procedures, by formatting posts' "created_at" timestamps. Maybe wrapping the cursor logic in functions like GetCursorFromPost and NewMaxCursor would be a better code design. However I don't see any additional future use cases, so it may be not worth the effort.

Reply:
This is reasonable, I think -- good call on now + 1 day!